gazelle/go.mod

```diff
@@ -9,6 +9,7 @@ require (
 	github.com/bmatcuk/doublestar v1.3.4
 	github.com/emirpasic/gods v1.18.1
 	github.com/ghodss/yaml v1.0.0
+	github.com/gobwas/glob v0.2.3
```

Comment:
This seems to be the most popular `glob` package, though I don't see any github activity in 6 years. Other options were:

+ https://pkg.go.dev/github.com/ryanuber/go-glob
+ https://pkg.go.dev/v.io/v23/glob

Reply:
How about `doublestar.Match`? It is used by `bazel-gazelle` and is already included in this `go.mod:9` too. 

```go
import (
	"fmt"

	"github.com/bmatcuk/doublestar/v4"
)

func matchesAnyGlob(s string, globs []string) bool {
	for _, g := range globs {
		if ok, _ := doublestar.Match(g, s); ok {
			return true
		}
	}
	return false
}

func main() {
	defaultGlob := []string{"*_test.py", "test_*.py"}
	fmt.Println(matchesAnyGlob("test_my_target.py", defaultGlob))
	fmt.Println(matchesAnyGlob("my_target_test.py", defaultGlob))

	customGlob := []string{"*_pytest.py", "pytest_*.py"}
	fmt.Println(matchesAnyGlob("pytest_upload.py", customGlob))
	fmt.Println(matchesAnyGlob("download_pytest.py", customGlob))
}
```